coreclient/src/user_profiles/display_name.rs

```diff
@@ -32,6 +33,39 @@ impl DisplayName {
             display_name: user_name.to_string(),
         }
     }
+
+    pub fn from_uuid(uuid: &Uuid) -> Self {
+        let animals = [
+            "Alpaca",
+            "Bear",
+            "Cat",
+            "Duck",
+            "Elephant",
+            "Fox",
+            "Giraffe",
+            "Hedgehog",
+            "Iguana",
+            "Jaguar",
+            "Kangaroo",
+            "Lion",
+            "Meerkat",
+            "Narwhal",
+            "Ostrich",
+            "Penguin",
+            "Quail",
+            "Rabbit",
+            "Squirrel",
+            "Toucan",
+            "Umbrellabird",
+            "Walrus",
+            "Yak",
+            "Zebra",
+        ];
+        let animal = animals[uuid.as_u128() as usize % animals.len()];
+        let number = uuid.as_u128() % 1000;
+        let display_name = format!("{} {}", animal, number);
```

Comment:
Should we prefix the animal and number by "anonymous" to indicate to the user that it's not their "real" display name?